storage/src/main/java/tech/pegasys/teku/storage/server/network/DatabaseNetwork.java

```diff
@@ -39,39 +42,81 @@ public class DatabaseNetwork {
   @VisibleForTesting
   final String depositContract;
 
+  @JsonProperty("deposit_chain_id")
+  @VisibleForTesting
+  final String depositChainId; // Can be null for non-Ephemery networks
+
+  static final Eth2NetworkConfiguration NETWORK_CONFIG =
+      Eth2NetworkConfiguration.builder(Eth2Network.EPHEMERY).build();
+
+  // Single constructor that handles both cases
   @JsonCreator
   DatabaseNetwork(
       @JsonProperty("fork_version") final String forkVersion,
-      @JsonProperty("deposit_contract") final String depositContract) {
+      @JsonProperty("deposit_contract") final String depositContract,
+      @JsonProperty("deposit_chain_id") final String depositChainId) {
     this.forkVersion = forkVersion;
     this.depositContract = depositContract;
+    this.depositChainId = depositChainId; // Can be null for non-Ephemery networks
   }
 
+  // Initialization logic
   public static DatabaseNetwork init(
-      final File source, final Bytes4 forkVersion, final Eth1Address depositContract)
+      final File source,
+      final Bytes4 forkVersion,
+      final Eth1Address depositContract,
+      final Optional<String> depositChainId)
       throws IOException {
     final String forkVersionString = forkVersion.toHexString().toLowerCase(Locale.ROOT);
     final String depositContractString = depositContract.toHexString().toLowerCase(Locale.ROOT);
     final ObjectMapper objectMapper =
         new ObjectMapper(new YAMLFactory().disable(WRITE_DOC_START_MARKER));
+
     if (source.exists()) {
       final DatabaseNetwork databaseNetwork =
           objectMapper.readerFor(DatabaseNetwork.class).readValue(source);
 
+      // Fork version validation
```

Comment:
we can remove these comments, they don't really add to the readability of the code...

Reply:
Alright, I will remove it.